Dockerfile

```diff
@@ -28,31 +28,35 @@ RUN apt-get update && apt-get install -y --no-install-recommends \
   wget \
   zlib1g-dev
 
-# Download packages
-RUN wget -qO- https://github.com/foosel/OctoPrint/archive/${version}.tar.gz | tar xz
-RUN wget -qO- https://github.com/jacksonliam/mjpg-streamer/archive/master.tar.gz | tar xz
 
-# Install mjpg-streamer
-WORKDIR /mjpg-streamer-master/mjpg-streamer-experimental
-RUN make
-RUN make install
+## Install mjpeg streamer
+WORKDIR /usr/app
+RUN git clone https://github.com/jacksonliam/mjpg-streamer.git .
+
+WORKDIR /usr/app/mjpg-streamer-experimental
+RUN make 
+RUN export LD_LIBRARY_PATH=.
+COPY ./mjpg_streamer_plugins/* /usr/app/mjpg-streamer-experimental/
```

Comment:
Thnaks for getting on that :) 

I made many tests to compile the rpi camera plugin of MJPG-STREAMER as it's a better solution as you suggested above. However, the command "RUN make" will fail to compile the raspi camera module because the dependecies are not satisfied.

Any idea on that are welcome. I found asomeone that compile the rpi camera plugin, but his starter image is resin/rpi-raspbian:stretch and he also install "libraspberrypi-dev" (see https://github.com/tdicola/pi_mjpeg_streamer). Not sure how to make it work with the base image "python:2.7-slim" while keeping the multiarch support.